changelog.txt

```diff
@@ -3,6 +3,7 @@
 = 1.25.2 - 2020-XX-XX =
 * Tweak - Add ZIP code validation to UPS(beta) signup form.
 * Fix   - Issue with printing labels in some iOS devices through Safari.
+* Tweak - Add message to explain exclusive
```

Comment:
I think the word tax might have gotten cut off here?

Reply:
Fixed!